magento/magento2
# Apply review suggestion to the media modules

### Description (*)
Removed Magento\MediaContentSynchronizationApi\Api\SynchronizerInterface
Removed try-catch block from \Magento\MediaGalleryMetadata\Model\File\ExtractMetadata::extractMetadata
Removed GetContentHashInterface
Removed DI configuration of Magento\MediaGalleryUi\Ui\Component\Listing\Filters\Options\UsedIn

### Fixed Issues (if relevant)
1. Fixes https://github.com/magento/adobe-stock-integration/issues/1725

### Preconditions
1. Enable Enhanced Media Gallery in Stores -> Configuration -> System -> Advanced
2. Several images are added from media gallery to Page, Product, Category, Block entities

### Testing (*)
1. Should be verified in two instances of media gallery: `Content -> Media Gallery`, Media Gallery opened from any wysiwyg
2. Verify Used in filter functionality
3. Verify Uploading the save image under a different name, saving image from AdobeStock twice (possible after removing the db record from `adobe_stock_asset` table) and applying Duplicates filter
4. Verify `media-content:sync` command

### Contribution checklist (*)
